src/game/auth.h

```diff
@@ -438,7 +438,16 @@ namespace auth
             conoutf("Updating master server");
             // the position of the 0 was used to signalize support for the global stats system, which has been removed
             // to retain compatibility, we just send a 0, pretending it was simply disabled by the administrator
-            requestmasterf("server %d %s %d %s 0 %s\n", serverport, *serverip ? serverip : "*", CUR_VERSION, escapestring(limitstring(G(serverdesc), MAXSDESCLEN+1)), escapestring(versionbranch));
+            char* advertisedIp;
```

Comment:
Style again: specify the pointer with `char* advertisedIp;` to make it consistent with the code below.